specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-11-15-preview/managedCassandra.json

```diff
@@ -774,657 +693,1717 @@
             },
-            "seedNodes": {
-              "readOnly": true,
-              "type": "array",
-              "items": {
-                "$ref": "#/definitions/SeedNode"
-              },
-              "description": "List of IP addresses of seed nodes in the managed data centers. These should be added to the seed node lists of all unmanaged nodes."
+            "sku": {
+              "type": "string",
+              "description": "Virtual Machine SKU used for data centers. Default value is Standard_DS14_v2"
             },
-            "hoursBetweenBackups": {
+            "diskSku": {
+              "type": "string",
+              "description": "Disk SKU used for data centers. Default value is P30."
+            },
+            "diskCapacity": {
               "type": "integer",
               "format": "int32",
-              "description": "Number of hours to wait between taking a backup of the cluster. To disable backups, set this property to 0."
+              "description": "Number of disk used for data centers. Default value is 4."
             },
-            "deallocated": {
+            "availabilityZone": {
               "type": "boolean",
-              "description": "Whether the cluster and associated data centers has been deallocated."
+              "description": "If the data center has Availability Zone support, apply it to the Virtual Machine ScaleSet that host the cassandra data center virtual machines."
             },
-            "cassandraAuditLoggingEnabled": {
+            "deallocated": {
               "type": "boolean",
-              "description": "Whether Cassandra audit logging is enabled"
+              "description": "Whether the data center has been deallocated."
+            },
+            "provisionError": {
+              "$ref": "#/definitions/CassandraError",
+              "description": "Error related to resource provisioning."
+            },
+            "authenticationMethodLdapProperties": {
+              "$ref": "#/definitions/AuthenticationMethodLdapProperties"
+            }
+          }
+        }
+      }
+    },
+    "CassandraClusterRepairPublicProperties": {
+      "type": "object",
+      "properties": {
+        "keyspace": {
+          "type": "string",
+          "description": "keyspace to be repaired."
+        },
+        "owner": {
+          "type": "string",
+          "description": "owner of the repair run."
+        },
+        "cause": {
+          "type": "string",
+          "description": "note on reason for repair run."
+        },
+        "tables": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "list of column famalies to be repaird. if empty whole keyspace will be repaired."
+        },
+        "segmentCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of segments in repair run"
+        },
```

Comment:
Is this a count of objects expose elsewhere through the API? If so does that list implicitly have a count?